tools/SetupFlow/DevHome.SetupFlow/ViewModels/Environments/ComputeSystemCardViewModel.cs

```diff
@@ -115,4 +117,18 @@ public void RemoveComputeSystemStateChangedHandler()
         ComputeSystemWrapper.StateChanged -= _computeSystemManager.OnComputeSystemStateChanged;
         _computeSystemManager.ComputeSystemStateChanged -= OnComputeSystemStateChanged;
     }
+
+    private string BuildAutomationName()
+    {
+        var stringBuilder = new StringBuilder();
+        stringBuilder.AppendLine(CultureInfo.CurrentCulture, $"{ComputeSystemTitle}");
```

Comment:
Should this be `InvariantCulture` instead since we're dealing with strings which won't have culture-specific behavior.